include/swift/Reflection/RuntimeInternals.h

```diff
@@ -94,28 +94,34 @@ struct StackAllocator {
 };
 
 template <typename Runtime>
-struct ActiveTaskStatus {
+struct ActiveTaskStatusWithEscalation {
+  uint32_t Flags;
+  uint32_t DrainLock[(sizeof(typename Runtime::StoredPointer) == 8) ? 1 : 2];
```

Comment:
This doesn't look right; it should be:

```
  uint32_t Flags;
  uint32_t DrainLock;
  typename Runtime::StoredPointer StatusRecords;
```

Right?

Reply:
Yes this is kind of a gross-ish hack that @mikeash and I came up with. It is meant to reflect the following layouts which are possible in 32 and 64 bit case of supporting priority escalation:

32 bit supporting priority escalation - ie arm64_32: 

```
uint32_t flags
uint32_t drainLock
uint32_t unused
typename Runtime::StoredPointer statusRecords
```

64 bit supporting priority escalation: 

```
uint32_t flags
uint32_t drainLock
typename Runtime::StoredPointer statusRecords
```

Reply:
Oh, I misread the code, sorry.